src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
+      <div class="calendar__day calendar__day--day-number--1"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
це має робити цикл а не ти ручками

Reply:
Так, в мене є цикл у стилях і він вносить контент у ці діви, та я думала, що потрібно, щоб кожен з цих календарів відрізнявся і за ім'ям)